### PR TITLE
New backup service api version

### PR DESCRIPTION
**Pull request by arpja**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>
- Added new API version 2020-12-01 to support selection of target availability zone in when restoring VMs.
### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet. Jan 2021
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet. Jan 2021
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by arpja** [comment]
Hi @lirenhe,
Please review the PR now, I have fixed the validation errors.

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/examples/AzureIaasVm/BackupPolicies_List.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "SwaggerTestRg",
+    "vaultName": "NetSDKTestRsVault",
+    "api-version": "2019-06-15",
```
Comment (varunkumta) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/examples/AzureIaasVm/BackupPolicies_List.json:6]: please update the api-version in all examples

**Comment by varunkumta** [comment]
>           "description": "Identifies the zone of the VM at the time of backup. Applicable only for zone-pinned Vms",

description seems to indicate this is a single value, but type is array. please update accordingly

---
Refers to: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/bms.json:5290 in aa5e825. [](commit_id = aa5e825dfc4c30376e345791533c1cd3670e23d3, deletion_comment = False)

**Comment by varunkumta** [comment]
>           "type": "array",

please add/update examples too

---
Refers to: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/bms.json:5291 in aa5e825. [](commit_id = aa5e825dfc4c30376e345791533c1cd3670e23d3, deletion_comment = False)

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/examples/AzureIaasVm/BackupPolicies_List.json (hunk 1 above)
Comment (arpja) [reply specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/examples/AzureIaasVm/BackupPolicies_List.json]: Updated

**Comment by arpja** [comment]
> > ```
> >       "type": "array",
> > ```
> 
> please add/update examples too
> 
> Refers to: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/bms.json:5291 in [aa5e825](https://github.com/Azure/azure-rest-api-specs/commit/aa5e825dfc4c30376e345791533c1cd3670e23d3). [](commit_id = [aa5e825](https://github.com/Azure/azure-rest-api-specs/commit/aa5e825dfc4c30376e345791533c1cd3670e23d3), deletion_comment = False)

Added zone in the example json

**Comment by arpja** [comment]
> > ```
> >       "description": "Identifies the zone of the VM at the time of backup. Applicable only for zone-pinned Vms",
> > ```
> 
> description seems to indicate this is a single value, but type is array. please update accordingly
> 
> Refers to: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-12-01/bms.json:5290 in [aa5e825](https://github.com/Azure/azure-rest-api-specs/commit/aa5e825dfc4c30376e345791533c1cd3670e23d3). [](commit_id = [aa5e825](https://github.com/Azure/azure-rest-api-specs/commit/aa5e825dfc4c30376e345791533c1cd3670e23d3), deletion_comment = False)

This is kept as 'array' since it is the convention followed by the Azure Compute team who own availability zones feature for VMs/Disks.
We have followed the same paradigm to avoid breaking changes incase there is a change in behavior from Compute team later.

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/examples/AzureIaasVm/ProtectionPolicies_CreateOrUpdate_Complex.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "resourceGroupName": "SwaggerTestRg",
     "vaultName": "NetSDKTestRsVault",
     "policyName": "testPolicy1",
-    "api-version": "2019-06-15",
+    "api-version": "2020-12-01",
```
Comment (varunkumta) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/examples/AzureIaasVm/ProtectionPolicies_CreateOrUpdate_Complex.json:7]: I meant update the api-versions for the files in the new folder, not existing folders

Comment (arpja) [reply specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-06-15/examples/AzureIaasVm/ProtectionPolicies_CreateOrUpdate_Complex.json]: My bad, reverted the commit and added a new one only for new API version

**Comment by ArcturusZhang** [comment]
Could we also include this new tag in those readme files for different languages of SDKs? For instance readme.go.md, readme.python.md, etc

**Comment by arpja** [comment]
> Could we also include this new tag in those readme files for different languages of SDKs? For instance readme.go.md, readme.python.md, etc

Added, please review now.
Thanks

**Review by ArcturusZhang — APPROVED** [review]
LGTM and please let me know when this is good to merge, thanks

**Pull request merged**